manifests/params.pp

```diff
@@ -106,6 +106,7 @@
         'php5-ldap',
         'php5-mysql',
         'php5-pgsql',
+        'mysql-client-core-5.5',
```

Comment:
Better yet use a module that provides this package.
I can recommend this one: [puppetlabs/mysql](https://forge.puppet.com/puppetlabs/mysql).